src/Synology.Api.Client/Apis/DownloadStation/Task/DownloadStationTaskEndpoint.cs

```diff
@@ -28,23 +31,28 @@ public DownloadStationTaskEndpoint(ISynologyHttpClient synologyHttpClient, IApiI
     /// </summary>
     /// <param name="request">Request parameters</param>
     /// <returns></returns>
-    public Task<BaseApiResponse> CreateAsync(DownloadStationTaskCreateRequest request)
+    public Task<DownloadStationTaskCreateResponse> CreateAsync(DownloadStationTaskCreateRequest request)
     {
         var queryParams = new Dictionary<string, string?>
         {
-            { "uri", request.Uri },
-            { "destination", request.Destination }
+            { "url", JsonSerializer.Serialize(request.Uri) },
```

Comment:
Do we really need to serialize all these values? It would be much more efficient to use the values directly.

Reply:
This param need to be `"htt....."` and not `http....`

Reply:
You don't think that using JSON serialization is overkill for this purpose? 

Can you try to use an extension method? For example:
```csharp
public static class StringExtensions
{
    /// <summary>
    /// Adds double quotes around the provided string.
    /// </summary>
    /// <param name="value">The string to wrap in double quotes.</param>
    /// <returns>The string wrapped in double quotes, or null if the input is null.</returns>
    public static string? AddDoubleQuotes(this string? value)
    {
        return value is null ? null : $"\"{value}\"";
    }
}
```

Usage:
```csharp
var queryParams = new Dictionary<string, string?>
{
    { "url", request.Uri.AddDoubleQuotes() },
};
```

Reply:
I did check again, as you may see, the first commit is from mid october and didn't remember everything I've done. 

First I think you was asking just for the url param but I did add the Serialize method on every single param.

It's not `url` but `type` that need a double quote before and after. 
Uri is now a true json array  `["link1", "link2", ...]` (before it was separate by comma).

I will prepare an other commit with some string Extensions for those case.